src/main/java/uk/gov/hmcts/reform/sscs/model/VenueDetails.java

```diff
@@ -21,4 +21,5 @@
     private String active;
     private String gapsVenName;
     private String comments;
+    private String epimsId;
```

Comment:
I don't think it will but @suetlaiHMCTS have you checked? Try starting up locally or on demo with this and see if you can create a case + send to ready to list. I'd ask QA for some input too

Reply:
@dattipoeHMCTS The above action all perform on my local env successfully. I update the screen capture into the ticket for reference.